content/api/ip-pools.apib

```diff
@@ -14,7 +14,7 @@ description: Manage IP pools, which allow you to group and assign dedicated IPs
     + signing_domain (string) - DKIM-verified signing domain
     + fbl_signing_domain (string) - FBL Signing Domain for the pool. Used only for emails to domains that require DKIM for FBL.  
     + ips (array[object]) - Sending IPs within the pool. Sending IP objects include fields `external_ip`,`hostname`, `auto_warmup_enabled` & `auto_warmup_stage`
-    + `auto_warmup_overflow_pool` (string) - IP Pool ID to be used as an overflow pool for auto IP warmup.
+    + `auto_warmup_overflow_pool` (string) - IP Pool ID to be used as an overflow pool during auto IP warmup. Can specify shared pool to overflow to the SparkPost Shared Pool if you have access.<br/>The auto_warmup_overflow_pool `shared pool` is not available for <a href="https://www.sparkpost.com/enterprise-email/"><span class="label label-warning"><strong>Enterprise</strong></span></a></strong> customers
```

Comment:
Can you use the backticks around the first mention of `shared pool` as well so it is clearer that this is the exact string that is needed?